library/errata_tool_product_version.py

```diff
@@ -242,6 +253,9 @@ def prepare_diff_data(before, after):
 def ensure_product_version(client, params, check_mode):
     result = {'changed': False, 'stdout_lines': []}
     params = {param: val for param, val in params.items() if val is not None}
+    # Edge case, this value is allowed to be None.
+    if 'suppress_push_request_jira' not in params:
+        params['suppress_push_request_jira'] = None
```

Comment:
With  `default=None`, all existing playbooks will start wiping the existing values out on the server.

We could pass a special constant like `omitted` here or something, but that's the first time I can remember seeing this in an ET REST API. This steers into some choppy edge-case waters.

For this PR, how about you just drop these two lines, and not make it possible to set this to `null` with Ansible yet? If it's really important to support this edge case, you can add it in another PR (with more testing).

Reply:
Good idea, I removed that edge case